### ISSUE TITLE
Adding validation options to form widget

### ISSUE THREAD

**Issue by mrx** [impresspages/ImpressPages#792]
I would like to add input validation and attribute options to the form widget. Each input would require a settings form. I imagine something similar to the radio and select options.

1) Do you think this would add too much clutter to the interface?

2) If developed as a separate plugin, forms could be saved and reused. I think this would be useful when building landing pages.

Your thoughts?

**Comment by maskas** [comment]
A separate more powerful form as a plugin would be my favorite. And in case it will become popular, it could be placed instead of default forms widget.

**Comment by mrx** [comment]
I agree. A few improvements to the form widget can solve 90%+ uses but complex forms and validation will need a better UI.
